### ISSUE TITLE
CountrySelector custom noResultMessage not used

### ISSUE THREAD

**Issue by emri99** [cedvdb/phone_form_field#50]
When a parameter `noResultMessage` is given it is not used at the end.

country_selector.dart:149-151
```dart
child: Text(
  PhoneFieldLocalization.of(context)?.noResultMessage ??
    widget.noResultMessage),
```

**Comment by emri99** [comment]
Also notice that `country_selector_test.dart` is still massively commented, I'm working on this issue as well to add test to avoid regression and restore commented tests ;)

**Comment by cedvdb** [comment]
fyi I released v4.0.0. I was very busy lately because I'm relocating. Hopefully everything is fixed.
